docs/030_user-guide/120_customization.md

```diff
@@ -34,9 +34,10 @@ The Watch configuration is a part of the Pepr module that allows you to watch fo
 
 | Field                        | Description                                                                                                      | Example Values                  |
 |------------------------------|------------------------------------------------------------------------------------------------------------------|---------------------------------|
-| `PEPR_RETRYMAX`              | The maximum number of times to retry the watch, the retry count is reset on success.                             | default: `"5"`                  |
+| `PEPR_RESYNC_FAILURE_MAX`    | The maximum number of times to fail on a relist interval before re-establishing the url and doing a relist    | default: `"5"`                |
```

Comment:
They are two different things but they are related. Once you hit the failureRetryMax, it does a `this.#watch` which does a list. https://github.com/defenseunicorns/kubernetes-fluent-client/blob/6a936fd89ed20de0f36c5d7ada96242be84cd92a/src/fluent/watch.ts#L471

Reply:
this.list being in watch https://github.com/defenseunicorns/kubernetes-fluent-client/blob/6a936fd89ed20de0f36c5d7ada96242be84cd92a/src/fluent/watch.ts#L360

Reply:
which is what relist does on the interval

Reply:
Righto.  Still seems kinda strange to define a "resync failure max" limit in terms of a failure of _relist_ intervals though (since the relists are a downstream consequence of the resync failures, no?  Would it be inaccurate to say instead:

"Them maximum number of times to fail on a **resync** interval before re-establishing the watch URL and doing a relist."